app/src/App.tsx

```diff
@@ -70,7 +70,7 @@ export const App = (): React.ReactElement => {
         <Route path='/register' page={RegistrationPage} />
         <Route path='/forgot-password' page={ForgotPasswordPage} />
         <Route path='/domains' page={DomainsPage} />
-        <Route path='/domains/create' page={CreateDomainPage} />
+        <Route path='/domains/create' page={DomainCreatePage} />
```

Comment:
why this renaming?

Reply:
so it follows the url more closely and also the pages appear together in the folders

Reply:
didnt actualyl realise a page existed for it already lol, happy to change back if you think its better the other way?

Reply:
nah, this is cool 👍

Reply:
the other way reads better but this follows nicely from the path

Reply:
ah next step is autogeneration 😛